src/ocean/text/convert/Float.d

```diff
@@ -36,8 +36,6 @@ import ocean.core.ExceptionDefinitions;
 static import tsm = core.stdc.math;
 static import Integer = ocean.text.convert.Integer_tango;
 
-version(UnitTest) import ocean.core.Test;
-
```

Comment:
Two points:
- `test` should be killed with fire, as the compiler should really be more clever about this
- If the first point is not addressed, another approach would be to move those tests to a separate module.

Reply:
I assume first statement has typo and is supposed to either mean "should not" or `assert`?

Reply:
I mean that the compiler should provide an helpful error message on assert error by default.
It could very well turn `assert(a == b)` into `druntimeAssert!("==")(a, b)` or anything of the like. But that's a bit OT.

Reply:
Yes, but won't make `assert` usable for testing. The fact that it throws `AssertError` is much bigger problem.

Reply:
(good suggestion about moving all tests to separate module though)

Reply:
Yes, good idea :+1: